### PR TITLE
Tailwindize incentive details (new calc only)

### PR DESCRIPTION
**Pull request by oyamauchi**
## Test Plan

Flip back and forth between this and prod, making sure they're
pixel-identical. Do this at all 3 layouts (w < 640, 640 <= w < 768,
and 768 <= x).

### CODE REVIEW

File: src/state-incentive-details.tsx [hunk 1]
```diff
@@ -360,29 +204,93 @@ const isIRARebate = (incentive: Incentive) =>
...
+    'font-bold',
+    'text-xsm',
+    'leading-tight',
+    'tracking-[0.03438rem]',
+    'uppercase',
+  ];
+  if (isWarning) {
+    classes.push(
+      'bg-yellow-200',
+      'text-red-500', // TODO should be #806c23
```
Comment (oyamauchi) [inline src/state-incentive-details.tsx:230]: This color doesn't seem to be in the design system; I'm not sure what to do about that. This text was originally spec'd as `#846f24`, which doesn't seem to be in the design system either (?) and does not have enough contrast with bg-yellow-200. I just put red-500 here as a placeholder, but it doesn't look good.

File: src/state-incentive-details.tsx [hunk 2]
```diff
@@ -506,8 +414,10 @@ const IncentiveGrid: FC<IncentiveGridProps> = ({
   emailSubmitter,
 }) => {
   return tabs.length > 0 ? (
-    <div className="grid-section" id={htmlId}>
-      <h2 className="grid-section__header">{heading}</h2>
+    <div className="min-w-52" id={htmlId}>
```
Comment (oyamauchi) [inline src/state-incentive-details.tsx:417]: This is 208px, although the original min-width of this was 200px. I think that's fine.

File: src/state-incentive-details.tsx (hunk 1 above)
Comment (RandomEtc) [reply src/state-incentive-details.tsx:230]: For parity with the old code you can specify arbitrary one-offs in tailwind like `bg-[#806c23]` and then we can follow up to see what should happen design-system wise.

Comment (RandomEtc) [reply src/state-incentive-details.tsx:230]: I see you already did that with the tracking value - I think that's fine for now.

**Comment by RandomEtc** [comment]
That's pretty satisfying honestly, and I didn't think our CSS was all that bad:

Screenshot 2024-01-12 at 2 31 33 PM

**Review by RandomEtc — APPROVED** [review]

**Pull request merged**